src/content/topics/integrations/Slack/notifications.mdx

```diff
@@ -56,6 +56,24 @@ To add a flag to a channel:
 
 ![A notification that a flag subscribed to the channel has had a status change.](../../images/slack-flag-change.png) 
 
+## Receiving alerts when flags are ready to remove 
+
+In the slack app, you can subscribe to notifications when it’s time to remove flags from code. Removing from code is a necessary step before archiving flags.
+
+Read about [deleting and archiving flags](https://docs.launchdarkly.com/home/managing-flags/flag-archive-delete) to learn how to keep your codebase clean. The Slack app makes it easy to receive notifications once flags are ready to remove.
+
+You will get served a “ready to remove” notification when any flags that you subscribe to become “inactive” or “launched” in status code.
+
+To set up a ready to remove flag alert:
+
+1. Open the channel where you want to get notifcations on flags that are ready to remove.
+2. Type `/launchdarkly susbcribe`and press **Enter**. The filter modal appears.
+3. Filter on the flag, project, environment, or tags where you want to be notified when flags have become "inactive" or "launched".
+4. Select "Ready for removal" from the Actions dropdown, which lets you only be notified on flags that are ready to remove and archive.
+5. Click **Filter**
```

Comment:
```suggestion
5. Click **Filter**.
```

Reply:
Minor, but I unresolved this -- I do think we should add punctuation since this is a (short yet) complete sentence.